docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
+| --config | string | Lets you [specify a path to a custom config file](#how-to-reference-a-custom-config-file). |
+| --extends | array | [Extends the existing config](#how-to-activate-rule-sets) by invoking rule sets that apply lots of rules at once. Possible values: `minimal`, `recommended`, `all`. Default: `recommended`. |
+| --format | string | [Applies formatting to the output](#how-to-format-linting-results). Possible values: `codeframe`, `stylish`, `json`, `checkstyle`. Default: `codeframe`. |
+| --generate-ignore-file | boolean | [Generates an ignore file](#how-to-ignore-specific-errors-and-warnings) which tells Redocly CLI to ignore certain warnings/errors when validating the definition. |
+| --help | boolean | [Displays a short help menu](#how-to-open-the-help), showing lint options. |
+| --max-problems | integer | [Truncates the output](#how-to-limit-issues-shown-in-the-output) to show only a specified number of errors and warnings. Default: `100`. |
+| --skip-preprocessor | array | [Ignores specific preprocessors](#how-to-skip-a-rule-or-preprocessor-during-linting). |
+| --skip_rule | array | [Ignores specific custom or built-in rules](#how-to-skip-a-rule-or-preprocessor-during-linting). |
+| --version | boolean | Adding `--version` to any command (not just `lint`) will show your current Redocly CLI version. |
 
-### Entrypoints
+## Usage examples
 
-The `lint` command behaves differently depending on how you pass entrypoints to it and whether the [configuration file](#custom-configuration-file) exists.
+### How to use entrypoints to check all or selected definitions
 
-#### Pass entrypoints directly
+##### Method 1: Pass entrypoints directly
+In this case, `lint` will validate the definitions passed in the command and ignore the `apis` section in `redocly.yaml` or any other customized config file you may be using.
 
 ```bash
-openapi lint openapi/openapi.yaml
+redocly lint openapi/openapi.yaml
 ```
 
-In this case, `lint` will validate the definition(s) passed to the command. The configuration file is ignored.
+The `entrypoints` argument can also use any glob format supported by your file system. For example, `redocly lint ./root-documents/*.yaml`.
 
-The `entrypoints` argument can also use any glob format supported by your file system. For example, `openapi lint ./root-documents/*.yaml`.
+##### Method 2: Pass entrypoints via the config file
 
-#### Pass entrypoints via configuration file
+The `apis` section in the Redocly configuration file (`redocly.yaml`) lets you configure one or more APIs. Each file path is specified (using the `root` property), but files are identified by their name and version in the format `name@version`.
 
-Instead of full paths, you can use names listed in the `apis` section of your Redocly configuration file as entrypoints.
+`@version` is optional, and when not provided, Redocly apps interpret it as `latest` by default. Every `name@version` combination listed in the `apis` section must be unique.
```

Comment:
I don't think this is appropriate for documentation because the version should be supplied. There is a lot of things people can do in an incomplete manner, but we don't call out explicitly that they can do it in an incomplete manner and it will still work (partially).

Reply:
But what if there are no versions? Are we saying we expect users to always express `name@version` regardless? How would that go for my one version...?

myapi@v1

Reply:
I think we don't want to encourage users to omit `version` because it can cause compatibility problems with the API registry. I don't know if you've tried this already, but when you're adding an API to the registry, it doesn't let you complete the action unless you specify the `version`. So in the light of that, it makes sense to sort of "steer" them in this happy-path direction where they just always specify the version and don't have to think about what would happen if they didn't have it.